### PR TITLE
docs: JK-10491: improve appearance of supported metrics reference section

### PR DESCRIPTION
**Pull request by jbyrne-redhat**
**Notes for reviewers**

- An HTML preview is available for review [here](http://file.emea.redhat.com/~jbyrne/JK-10491/Monitoring%20metrics%20in%20OpenShift%20Streams%20for%20Apache%20Kafka.html)

### CODE REVIEW

**Review by AedinC — APPROVED** [review]
Great updates John- doc looks much improved and easier to read. Just a few very minor comments.

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 1]
```diff
@@ -103,58 +103,64 @@ As a developer or administrator, you can view metrics in {product-long-kafka} to
...
 
-* `kafka_server_brokertopicmetrics_bytes_in_total`: Number of incoming bytes to topics in the instance.
-
-* `kafka_server_brokertopicmetrics_bytes_out_total`: Number of outgoing bytes from topics in the instance.
-
-* `kafka_server_brokertopicmetrics_messages_in_total`: Number of messages per second received by one or more topics in the instance.
+`kafka_server_brokertopicmetrics_bytes_out_total`:: Number of outgoing bytes from topics in the instance.
 
-* `kafka_topic:kafka_server_brokertopicmetrics_bytes_in_total:rate5m`: Number of incoming bytes to topics in the instance in the last five minutes.
+`kafka_server_brokertopicmetrics_messages_in_total`:: Number of messages per second received by one or more topics in the instance.
```
[the remark this answers is not in the document]
Comment (jbyrne-redhat) [reply docs/kafka/metrics-monitoring-kafka/README.adoc:154]: You raise a very good point, @AedinC . I didn't write the metrics descriptions, and now that you've made me read these two more closely, I'm quite confused. 

The metric with 'total' in its name is described more like a rate. And the metric with a 'rate' in its name sounds more like a total :laughing: 

I'll look into these further.

[the remark this answers is not in the document]
Comment (jbyrne-redhat) [reply docs/kafka/metrics-monitoring-kafka/README.adoc:154]: Discussed with Ger Ryan the issue of names/descriptions of some existing metrics appearing to disagree. Per https://docs.google.com/spreadsheets/d/13I-sEby5xgN4RfX8MmLRagP9tDr3KA9I_N_k5sioPQ4/edit#gid=0, Jameel Briones will be leading a future effort to improve our set of supported metrics. One update, for example, will be removing the word 'total' from the names of metrics that are actually rates. For example, `kafka_namespace:haproxy_server_bytes_in_total:rate5m` --> `kas_instance_incoming_bytes_rate5m`. 

As part of that effort, I'm planning to create a new CCS task to work with Jameel on a complete review of all of our metrics descriptions. As a result, though, I'm not going to use this PR to make any meaningful updates to existing descriptions, apart from the more minor tweaks. Will leave the bigger changes to that dedicated review.

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 2]
```diff
@@ -103,58 +103,64 @@ As a developer or administrator, you can view metrics in {product-long-kafka} to
...
 
-* `kafka_topic:kafka_server_brokertopicmetrics_bytes_out_total:rate5m`: Number of outgoing bytes from topics in the instance in the last five minutes.
+`kafka_topic:kafka_server_brokertopicmetrics_bytes_in_total:rate5m`:: Number of incoming bytes to topics in the instance in the last five minutes.
```
[the remark this answers is not in the document]
Comment (jbyrne-redhat) [reply docs/kafka/metrics-monitoring-kafka/README.adoc:156]: See https://github.com/redhat-developer/app-services-guides/pull/627#discussion_r1090418149.

**Pull request merged**